vendor/brave-ios/Ledger/BATBraveLedger.mm

```diff
@@ -417,21 +418,23 @@ - (void)publisherActivityFromURL:(NSURL *)URL
 
 - (void)mediaPublisherInfoForMediaKey:(NSString *)mediaKey completion:(void (^)(BATPublisherInfo * _Nullable))completion
 {
-  ledger->GetMediaPublisherInfo(std::string(mediaKey.UTF8String), ^(ledger::Result result, ledger::PublisherInfoPtr info) {
-    if (result == ledger::LEDGER_OK) {
-      const auto* publisherInfo = info.get();
-      if (publisherInfo != nullptr) {
-        completion([[BATPublisherInfo alloc] initWithPublisherInfo:*publisherInfo]);
-      }
-    } else {
-      completion(nil);
-    }
-  });
+  // Q: why is this needed?
```

Comment:
It's not, we can remove the entire method here (and it's declaration in the header file)